server.spec.js

```diff
@@ -0,0 +1,158 @@
+const request = require('supertest');
+const server = require('./server.js');
+const games = require('./db');
+
+describe('server.js', () => {
+  describe('/ route', () => {
+    it('should return a status code of 200', async () => {
+      let response = await request(server).get('/');
+      expect(response.status).toBe(200);
+    });
+  });
+  describe('/games route', () => {
+    describe('GET', () => {
+      it('should return a status code of 200', async () => {
+        let response = await request(server).get('/games');
+        expect(response.status).toBe(200);
+      });
+      it('should return JSON', async () => {
+        let response = await request(server).get('/games');
+        expect(response.type).toBe('application/json');
+      });
+      it('should always return an array', async () => {
+        let response = await request(server).get('/games');
+        expect(Array.isArray(response.body)).toBe(true);
+      });
```

Comment:
There was a request in the instructions to test for when there are no saved games. How would you go about implementing that?